pkg/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.nupkgproj

```diff
@@ -4,7 +4,7 @@
     <Authors>The TensorFlow Authors</Authors>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageDescription>$(MSBuildProjectName) contains the TensorFlow C library version $(TensorFlowVersion) redistributed as a NuGet package.</PackageDescription>
-    <PackageLicenseUrl>https://github.com/tensorflow/tensorflow/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
```

Comment:
does it get downloaded with .txt extension ?

(in the .proj file I do not see the .txt extension)

Reply:
The `.txt` extension gets added when the file gets copied:

https://github.com/dotnet/machinelearning/blob/b89ce70824885d8c3a57dc62d6c88cbae9fd8a60/src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.proj#L106-L108

Reply:
awesome. thanks for the info.

---
In reply to: [248750410](https://github.com/dotnet/machinelearning/pull/2161#discussion_r248750410) [](ancestors = 248750410)